src/MuTalk-Tests/AllTestsMethodsRunningTestSelectionStrategyTest.class.st

```diff
@@ -0,0 +1,36 @@
+Class {
+	#name : 'AllTestsMethodsRunningTestSelectionStrategyTest',
+	#superclass : 'TestSelectionStrategyTest',
+	#category : 'MuTalk-Tests',
+	#package : 'MuTalk-Tests'
+}
+
+{ #category : 'testing' }
+AllTestsMethodsRunningTestSelectionStrategyTest class >> isAbstract [
+	^ false
+]
+
+{ #category : 'asserting' }
+AllTestsMethodsRunningTestSelectionStrategyTest >> assertForAllMutants [
+	self assert: testCases size equals: testCasesUsed size
+]
+
+{ #category : 'asserting' }
+AllTestsMethodsRunningTestSelectionStrategyTest >> assertMutantInCoverageOfTestCase [
+	self assertForAllMutants
+]
+
+{ #category : 'asserting' }
+AllTestsMethodsRunningTestSelectionStrategyTest >> assertMutantNotInCoverage [
+	self assertForAllMutants
+]
+
+{ #category : 'asserting' }
+AllTestsMethodsRunningTestSelectionStrategyTest >> assertMutantNotInCoverageOfTest [
+	self assertForAllMutants
+]
```

Comment:
This test class looks really strange. All tests are doing the same! What's their purpose?
They just take 4x longer to run!